MAINTAINERS.md

```diff
@@ -84,3 +84,177 @@ Submit a PR with the changes and request a core team review.
 
 Update the list of active branches in the `hacking/tagger/tag.py` script on the `devel` branch.
 Add the new stable branch and remove the lowest version from the `DEFAULT_ACTIVE_BRANCHES` tuple.
+
+## Updating the package doc builds repository
+
+The `ansible-community/package-doc-builds` repository holds the deployed output of package doc builds.
+For each version of the Ansible package that is currently supported, there needs to be a corresponding branch in the `ansible-community/package-doc-builds` repository.
+
+Before the docs for new package version can be built on Read the Docs, someone needs to create a branch as follows:
+
+```bash
+# Make sure your checkout is up to date.
+git fetch upstream
+
+# Create the new package version branch against the devel branch.
+git checkout -b 11 upstream/devel
+
+# Push the new package version branch to the repository.
+git push upstream 11
+```
+
+## Updating the package docs build workflow
+
+The build workflow lists the Ansible package versions as input options, for example:
+
+```yaml
+options:
+- devel
+- '11'
+- '10'
+- '9'
+```
+
+After creating a new branch for the Ansible package version in the `ansible-community/package-doc-builds` repository, you should check that the build workflow lists that version as an option.
```

Comment:
```suggestion
After creating a new branch for the Ansible package version in the `ansible-community/package-doc-builds` repository, you should check that the `.github/workflows/build-package-docs.yaml` lists that version as an option.
```

Reply:
I don't think we need to be specific about the name of the workflow file here. That seems to interrupt readability imo.